pages/index.js

```diff
@@ -16,9 +16,10 @@ import SalesforceLogo from '../components/logos/Salesforce'
 import SapLogo from '../components/logos/SAP'
 import Testimonial from '../components/Testimonial'
 import BlogPostItem from '../components/navigation/BlogPostItem'
+import meetingsData from '../meetings.json'
+import NewCalendar from '../components/NewCalendar'
```

Comment:
I'd not call it **New**Calendar. This name is not going to age well 😄 Maybe `DynamicCalendar`?

Reply:
Done.